### ISSUE TITLE
Add size bead support to Excel interface

### ISSUE THREAD

**Issue by jakebeal** [TASBE/TASBEFlowAnalytics#357]
Now that we can do FSC calibration with size beads (#158), we should support its use from the Excel interface as well.

**Comment by coverney** [comment]
Todo (writing it here in case I forget): update ExcelREADME

**Comment by coverney** [comment]
I just wanted to double check. Does this line ```CM=set_um_channel_name(CM, 'FSC-A');``` stay consistent whenever size beads are used?

**Comment by coverney** [comment]
Also, should the size bead beadModel and rangeMin parameters have separate cells in the template?

**Comment by coverney** [comment]
Currently, I have three parameters for size beads located in the "Additional Preferences for Color Model" of the Calibration tab: size bead file, size bead bead model, and size bead range min. Should I add or remove any of them?

**Comment by coverney** [comment]
Updates: I tried implementing the size beads feature, and I thought of the following questions:

1. Currently, I am manually inputting this code snippet whenever the size beads feature is being used
```
channels{4} = Channel('FSC-A', 488, 488, 10);
channels{4} = setPrintName(channels{4}, 'FSC');
channels{4} = setLineSpec(channels{4}, 'k');
```
Should I have the user input this information into the excel interface? If so, where would be a good place for that? I was thinking of creating a subsection in the "Fluorochromes" section. 

2. I am encountering the following error while trying to create the colormodel: ```Cannot translate
unprocessed channel```. This is because the FSC-A channel is set to be unprocessed. I was wondering if you ever encountered this error before.

**Comment by coverney** [comment]
How the size bead configs are currently placed in the excel interface:
![template](https://user-images.githubusercontent.com/25420615/48138667-ca1f5400-e272-11e8-802f-66c2004e3434.PNG)

**Comment by coverney** [comment]
I also updated the "Optional Settings" tab with the new size bead configs

**Comment by jakebeal** [comment]
> Does this line CM=set_um_channel_name(CM, 'FSC-A'); stay consistent whenever size beads are used?

It's usually this, but sometimes has to be changed because they've named the channel something else in their instrument.  Because of this and the other preferences that you'll want to set, I think it would be better to give size beads their own lines just like the "Rainbow Beads" lines --- every preference that you can set for rainbow beads you might also set for size beads.

> Should I have the user input this information into the excel interface? If so, where would be a good place for that? I was thinking of creating a subsection in the "Fluorochromes" section.

I think that's a good idea.  Perhaps "Fluorochromes" and "Non-fluorescence Channels", and all of the non-fluorescence channels can be set to unprocessed.

> I am encountering the following error while trying to create the colormodel: Cannot translate unprocessed channel. This is because the FSC-A channel is set to be unprocessed. I was wondering if you ever encountered this error before.

It should be unprocessed, and shouldn't be trying to make a translation model.  If it's trying to make a translation model, maybe your code is including the unprocessed channels when it's expanding "all" in the "relevant channels" cell for color translation?  If so, that's probably what's going wrong.

**Comment by coverney** [comment]
I modified the Excel interface. How does this version look?
![interface](https://user-images.githubusercontent.com/25420615/48486127-9ef3b200-e7e8-11e8-87b0-ce89e7fbaa93.PNG)

**Comment by jakebeal** [comment]
Looks great!

**Comment by coverney** [comment]
Awesome! I was also wondering what ```set_um_channel_name``` does. I tried running make_color_model_excel and got this interesting error that I am trying to make sense of:
```Cannot find an exact (case-sensitive) match for 'um_channel_AU_to_um'The closest match is: uM_channel_AU_to_uM in C```

**Comment by coverney** [comment]
I think I figure out the problem. It turns out that the ```um_channel_AU_to_um``` function is in a file titled ```uM_channel_AU_to_uM```. Should I change the function call in ```readfcs_compensated_ERF``` or change the file name?

**Comment by jakebeal** [comment]
`set_um_channel_name` is the size-bead equivalent of `set_ERF_channel_name`.
You should change the file-name: the "m" should be lower-case because it's short for "meters"

**Comment by coverney** [comment]
I got everything working. Now, I just need to modify the excel tests and then I should be ready to push

**Comment by coverney** [comment]
I tried testing everything, and it failed because it still thinks the file is named ```uM_channel_AU_to_uM.m``` even though I changed it. I will try to rename the file in GitHub, and if that doesn't work, I will change how the function is called.

**Comment by coverney** [comment]
Nevermind, it was actually working! I will wrap this issue up and submit a pull request.

**Comment by jakebeal** [comment]
Fixed by #365
